components/Themes/examples/ThemerCustomSchemePickerController.m

```diff
@@ -181,7 +181,8 @@ - (void)applyColorScheme:(id<MDCColorScheme>)colorScheme {
   // Apply color scheme to material design components using component themers.
   [MDCActivityIndicatorColorThemer applyColorScheme:colorScheme
                                 toActivityIndicator:[MDCActivityIndicator appearance]];
-  [MDCAlertColorThemer applyColorScheme:colorScheme];
+  [MDCAlertColorThemer applyColorScheme:colorScheme
+                     toAlertContentView:[MDCAlertContentView appearance]];
```

Comment:
Allow the CustomTheme picker to set custom text colors so we can verify the themer works as expected?